.secrets.baseline

```diff
@@ -127,10 +127,15 @@
         "(?i)^apikey$",
         "(?i)^test_api_key$",
         "(?i)^test_basic_auth_secret$",
-        "(?i)^test_authorisation_code$"
+        "(?i)^test_authorisation_code$",
+        "E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM",
+        "KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04",
+        "RBXnILIdExUEWUJMlYeD6agE8u9gGgA3InKrd5TKhhY",
+        "kKtt9v_xq9oqvv5_E8AHcV77IYQfyNwaTQyTYxdO_UM",
+        "vyapkvJXLwpYRJjbkQD99V2gcPEUKrO3dwjcAA9TPkLucQEZvYZvb7-wfSHxlvJlJcdS20r5PKKmqdPeW3Y4ir3WsVVeiht2iOZUreUO5O3V3o7ImvEjPS_2_ZKMHCwUf51a6WGOaDjO87OX_bluV2dp01n-E3kiIl6RmWCVywjn13fX3jsX0LMCM_bt3HofJqiYhhNymEwh39oR_D7EE5sLUii2XvpTYPa6L_uPwdKa4vRl4h4owrWEJaJifMorGcvqhCK1JOHqgknN_3cb_ns9Px6ynQCeFXvBDJy4q71clkBq_EZs5227Y1S222wXIwUYN8w5YORQe3M-pCIh1Q"
```

Comment:
These are the high-entropy components of the test keys - this would probably let us remove a bunch of allowlist comments, but there's no real urgency to do so.